funcx_endpoint/funcx_endpoint/version.py

```diff
@@ -1,6 +1,6 @@
 # single source of truth for package version,
 # see https://packaging.python.org/en/latest/single_source_version/
-__version__ = "1.0.7"
+__version__ = "1.0.8a"
```

Comment:
I think we usually use 'a0'?  https://pypi.org/project/funcx-endpoint/1.0.6a0/

Reply:
Updated